### PR TITLE
Separate out Slack Notify Messages to remove CMS Dependency

### PR DESCRIPTION
**Pull request by ariperez**
closes https://github.com/department-of-veterans-affairs/va.gov-cms/issues/13983

### CODE REVIEW

File: .github/actionlint.yml
Comment (ndouglas) [inline .github/actionlint.yml:1]: **Question**: What does this do? 🧐

File: .github/workflows/content-release.yml [hunk 1]
```diff
@@ -18,7 +18,8 @@ env:
   DEPLOY_BUCKET: content.www.va.gov
   DRUPAL_ADDRESS: https://prod.cms.va.gov
   INSTANCE_TYPE: m6i.4xlarge
-  MAXIMUM_HEAP: 5000
+  MAXIMUM_HEAP:
+    5000
```
Comment (ndouglas) [inline .github/workflows/content-release.yml:22]: **Question**: Is this change intentional?

File: .github/workflows/content-release.yml [hunk 2]
```diff
@@ -66,8 +67,8 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
-  notify-start:
-    name: Notify Start
+  notify-start-slack:
+    name: Notify Start (Slack)
```
Comment (ndouglas) [inline .github/workflows/content-release.yml:71]: **Praise**: 👍🏻

File: .github/workflows/content-release.yml [hunk 3]
```diff
@@ -531,6 +530,22 @@ jobs:
...
+    runs-on: [self-hosted, asg]
+    if: |
+      (failure() && needs.deploy.result != 'success')
+    needs: deploy
+    steps:
+      - name: Notify Slack
+        uses: department-of-veterans-affairs/platform-release-tools-actions/slack-notify@8c496a4b0c9158d18edcd9be8722ed0f79e8c5b4 #
+        continue-on-error: true
+        with:
+          payload: '{"attachments": [{"color": "#2EB67D","blocks": [{"type": "section","text": {"type": "mrkdwn","text": ":bangbang: <!subteam^S010U41C30V|cms-helpdesk> Content release using ${{ needs.validate-build-status.outputs.TAG }} has failed."}}]}]}'
```
Comment (ndouglas) [inline .github/workflows/content-release.yml:544]: **Suggestion (non-blocking)**: A fallback value for the CMS version to avoid this:

Screenshot 2023-08-28 at 2 52 14 PM

Again, non-blocking.

**Review by ndouglas — DISMISSED** [review]
This looks great to me, just a couple questions for my own education 👨🏻‍🎓  and one mild suggestion.

**Review by ndouglas — COMMENTED** [review]
Oops, should've just commented first, because of the MAXIMUM_HEAP change.

File: .github/actionlint.yml
Comment (ariperez) [reply .github/actionlint.yml:1]: it helps lint github action files; was experencing a warning for 'asg', can remove if you hate it...

https://github.com/wearerequired/lint-action

Comment (ndouglas) [reply .github/actionlint.yml:1]: Nah, was just curious 🙂

File: .github/workflows/content-release.yml (hunk 1 above)
Comment (ariperez) [reply .github/workflows/content-release.yml]: this was a formatting line length thing

Comment (ariperez) [reply .github/workflows/content-release.yml]: I pushed a change so the comment post-fixed didn't cause a new line

File: .github/workflows/content-release.yml (hunk 3 above)
Comment (ariperez) [reply .github/workflows/content-release.yml:543]: hmm, should have caught this, any suggestions?

Comment (ndouglas) [reply .github/workflows/content-release.yml:543]: suppose you could just do `${{ needs.validate-build-status.outputs.TAG || 'an unknown version' }}` 🤷🏻‍♂️

**Review by ndouglas — APPROVED** [review]
🚀 This looks really good.

**Review by powellkerry — APPROVED** [review]
